src/test/java/tests/elements/factory/ElementFactoryTests.java

```diff
@@ -148,51 +148,50 @@ public void shouldThrowIllegalArgumentExceptionInGetCustomElementElementTypeIsUn
     }
 
     @Test
     public void shouldBePossibleToFindChildElementWithSpecificNameUsingSupplier() {
         IElement parent = getParentElement();
         String name = "123";
-        Assert.assertEquals(customFactory().findChildElement(parent, CalculatorWindow.getEqualsButton(), name, CustomElement::new).getName(), name);
+        Assert.assertEquals(customFactory().findChildElement(parent, CalculatorWindow.getEqualsButtonLoc(), name, CustomElement::new).getName(), name);
     }
 
     @Test
     public void shouldBePossibleToFindChildElementViaCustomFactoryUsingSupplier() {
         IElement parent = getParentElement();
-        Assert.assertNotNull(customFactory().findChildElement(parent, CalculatorWindow.getEqualsButton(), CustomElement::new));
+        Assert.assertNotNull(customFactory().findChildElement(parent, CalculatorWindow.getEqualsButtonLoc(), CustomElement::new));
     }
 
     @Test
     public void shouldBePossibleToFindChildElementViaDefaultFactoryUsingImplementation() {
         IElement parent = defaultFactory().getCustomElement(CustomElement.class, CalculatorWindow.getWindowLocator(), "window");
-        Assert.assertNotNull(defaultFactory().findChildElement(parent, CalculatorWindow.getEqualsButton(), CustomElement.class));
+        Assert.assertNotNull(defaultFactory().findChildElement(parent, CalculatorWindow.getEqualsButtonLoc(), CustomElement.class));
     }
 
     @Test
     public void shouldBePossibleToFindChildElementWithCustomState() {
         IElement parent = getParentElement();
-        Assert.assertEquals(
-                defaultFactory().findChildElement(parent, CalculatorWindow.getEqualsButton(), CustomElement.class,
-                        ElementState.EXISTS_IN_ANY_STATE).getState(),
-                ElementState.EXISTS_IN_ANY_STATE);
+        Assert.assertNotNull(
+                defaultFactory().findChildElement(parent, CalculatorWindow.getEqualsButtonLoc(), CustomElement.class,
+                        ElementState.EXISTS_IN_ANY_STATE).getElement(),
+                "Element should be found");
     }
 
     @Test
     public void shouldSetCorrectParametersWhenGettingElement() {
         String name = "1some2";
-        CustomElement element = defaultFactory().getCustomElement(CustomElement::new, CalculatorWindow.getEqualsButton(), name, ElementState.EXISTS_IN_ANY_STATE);
-        Assert.assertEquals(element.getLocator(), CalculatorWindow.getEqualsButton());
+        CustomElement element = defaultFactory().getCustomElement(CustomElement::new, CalculatorWindow.getEqualsButtonLoc(), name, ElementState.EXISTS_IN_ANY_STATE);
+        Assert.assertEquals(element.getLocator(), CalculatorWindow.getEqualsButtonLoc());
         Assert.assertEquals(element.getName(), name);
-        Assert.assertEquals(element.getState(), ElementState.EXISTS_IN_ANY_STATE);
```

Comment:
oh no, I believe we still need to test the state. please restore the getState() method in test classes

Reply:
or could be checked via reflection ofc

Reply:
check, but it wasn't possible

Reply:
fixed